go.mod

```diff
@@ -2,19 +2,24 @@ module github.com/Windscribe/go-vhost
 
 go 1.19
 
-require github.com/quic-go/quic-go v0.34.0
+require (
+	github.com/cuonglm/quicsni v0.0.1
```

Comment:
should reference the Windscribe repo

Reply:
See the `replace` directive below.

Reply:
whats the benefit of using a replace here?

Reply:
This is how go modules world. The modules name is not replaced, but "where" to fetch the modules. The `replace` tells `go` whenever doing something with module `github.com/cuonglm/quicsni`, go to `github.con/Windscribe/quicsni` and fetch the module content there.

If you do really want to refer to Windscribe only, we can rename the module in Windscribe repo and use it as import path here.

Reply:
Anyway, I changed it to use Windscribe/quicsni, PTAL!